roaring/README.md

```diff
@@ -0,0 +1,30 @@
+# The Fuzzer
+
+For complete documentation on go-fuzz, please see: https://github.com/dvyukov/go-fuzz
+
+The fuzzer in relation to the roaring package checks the `Bitmap.UnmarshalBinary` function found in `roaring.go`. In order to use the fuzzer, you can follow these steps:
+
+`cd $GOPATH/src/github.com/pilosa/pilosa/roaring`
```

Comment:
This whole instruction section is a good example of something that could be incorporated into a makefile. Maybe as an "install-gofuzz" target, following the convention set there. This could be a new ticket though.